### ISSUE TITLE
Add more text justifications

### ISSUE THREAD

**Issue by ExcaliburZero** [vincenthz/hs-cli#14]
Currently the `Justify` datatype and `justify` function only have support for left and right justification. It would be useful to also have support for center and "justified" justifications.

```
>>> justify JustifyCenter 35 "Lorem ipsum dolor sit amet"
"     Lorem ipsum dolor sit amet    "

>>> justify JustifyJustified 35 "Lorem ipsum dolor sit amet"
"Lorem    ipsum   dolor   sit   amet"
```

**Comment by vincenthz** [comment]
I don't really have time to add this myself, but this would be a more than welcome addition

**Comment by ExcaliburZero** [comment]
I've finished writing the handling for centered and justified text. However, I do have one question.

It would be good if I could also add some tests for the `justify` function, as it is an easily testable pure function. However, the current testing setup seems to focus on testing the command option parser. How would I best go about adding in the tests of the `justify` function?

Currently, I have included some examples of the function usage in the documentation which can be tested with doctest.
